docs/contracts.rst

```diff
@@ -571,8 +572,8 @@ In addition to the list of state modifying statements explained above, the follo
   not do state-changing operations, but it cannot check that the contract that will be called
   at runtime is actually of that type.
 
-.. warning::
-  Before version 0.5.0 the compiler did not enforce that ``view`` is not writing the state.
+.. note::
+  Before version 0.5.0 the compiler did not enforce that ``pure`` is not reading the state.
```

Comment:
It actually didn't enforce view not writing the state, because of backwards compatibility.

Reply:
Right, pure did not exist at some point, but did we actually enforce it from day one?

Reply:
Pure was enforced, view wasn't.